src/containers/CategoryFilter.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Form from 'react-bootstrap/Form';
+import { changeFilter } from '../actions/index';
+
```

Comment:
this is a presentational component, please relocate it inside `src/components/` folder
[see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/presentationalCategoryFilter.png)]

Reply:
Perfect